help/include/mute-unmute-intro.md

```diff
@@ -0,0 +1,24 @@
+To avoid receiving notifications messages you are not interested in, Zulip lets
+you mute topics and streams. Muting a stream effectively mutes all topics in
```

Comment:
I ran a search for "Zulip lets you..." and noticed that in general we put it at the beginning of the intro paragraphs. I think the phrasing is more direct and it would read a little better:

```suggestion
Zulip lets you mute topics and streams to avoid receiving notifications messages
you are not interested in. Muting a stream effectively mutes all topics in
```